app/main/forms/suppliers.py

```diff
@@ -139,3 +139,40 @@ class CompanyOrganisationSizeForm(Form):
     organisation_size = RadioField('Organisation size',
                                    validators=[InputRequired(message="You must choose an organisation size.")],
                                    choices=[(o['value'], o['label']) for o in OPTIONS])
+
+
+class CompanyTradingStatusForm(Form):
+    OPTIONS = [
+        {
+            'value': 'limited company (LTC)',
+            'label': 'limited company (LTC)',
```

Comment:
"limited company" should be "LTD" rather than "LTC", according to the ticket.

Reply:
Correct, updated.